qmlti/lib.php

```diff
@@ -293,11 +307,16 @@ function init_db($db) {
       if ($ok) {
         $sql = 'CREATE TABLE IF NOT EXISTS ' . TABLE_PREFIX . LTI_Data_Connector::RESOURCE_LINK_TABLE_NAME . ' ' .
                '(consumer_key VARCHAR(50) NOT NULL DEFAULT \'\',' .
-               ' context_id VARCHAR(100),' .
+               ' lti_context_id VARCHAR(100),' .
+               ' lti_resource_id VARCHAR(255),' .
+               ' title VARCHAR(255),' .
                ' settings TEXT,' .
+               ' primary_consumer_key VARCHAR(255),' .
+               ' primary_context_key VARCHAR(255),' .
+               ' share_approved INT,' .
                ' created DATETIME,' .
                ' updated DATETIME, ' .
-               'PRIMARY KEY (consumer_key, context_id))';
+               'PRIMARY KEY (consumer_key, lti_context_id))';
```

Comment:
Again not 100% sure of the purpose of the table but I would have expected lti_resource_id to be part of the PK?